tests/kernels/test_attention.py

```diff
@@ -364,10 +372,10 @@ def ref_multi_query_kv_attention(
     return torch.cat(ref_outputs, dim=0)
 
 
-# TODO(woosuk): Add tests for USE_ALIBI=True.
 @pytest.mark.parametrize("num_seqs", NUM_PREFILL_SEQS)
 @pytest.mark.parametrize("num_heads", NUM_HEADS)
 @pytest.mark.parametrize("head_size", HEAD_SIZES)
+@pytest.mark.parametrize("use_alibi", USE_ALIBI)
```

Comment:
This does increase the test time by 2x because it's testing all the possible combinations. Can we make a separate test case for alibi and test a (very) smalls subset of shapes?

Reply:
sure thing